src/rpc/util.cpp

```diff
@@ -203,6 +199,10 @@ CTxDestination AddAndGetMultisigDestination(const int required, const std::vecto
         }
     }
 
+    if (type == OutputType::LEGACY && script_out.size() > MAX_SCRIPT_ELEMENT_SIZE) {
```

Comment:
Shouldn't this be `OutputType::P2SH_SEGWIT`?

Reply:
No, the limit only applies to raw P2SH not to embedded witness programs